sitemap_index.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<sitemapindex xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+   <sitemap>
+      <loc>https://www.min.io/docs/linux/sitemap.xml</loc>
```

Comment:
We might as well update these to what we have in #minio-docs pending AB's final sign-off

Reply:
Updated